incubator/chartmuseum/templates/deployment.yaml

```diff
@@ -2,48 +2,67 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: {{ include "common.fullname" . }}
+  annotations:
+{{ toYaml .Values.deployment.annotations | indent 4 }}
   labels:
 {{ include "common.labels.standard" . | indent 4 }}
 spec:
   replicas: {{ .Values.replicaCount }}
   template:
     metadata:
       name: {{ include "common.fullname" . }}
+      annotations:
+{{ toYaml .Values.replica.annotations | indent 8 }}
       labels:
         app: {{ template "common.name" . }}
         release: {{ .Release.Name | quote }}
     spec:
-      volumes:
-      - name: storage-volume
-        emptyDir: {}
       containers:
       - name: {{ .Chart.Name }}
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+{{- range $name, $value := .Values.env.open }}
+{{- if not (empty $value) }}
+        - name: {{ $name | quote }}
+          value: {{ $value | quote }}
+{{- end }}
+{{- end }}
+{{- $secret_name := include "common.fullname" . }}
+{{- range $name, $value := .Values.env.secret }}
+{{- if not ( empty $value) }}
+        - name: {{ $name | quote }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ $secret_name }}
+              key: {{ $name | quote }}
+{{- end }}
+{{- end }}
         args:
         - --port={{ .Values.service.internalPort }}
-        - --debug
-        - --storage=local
+{{- if eq .Values.env.open.STORAGE "local" }}
         - --storage-local-rootdir=/storage
```

Comment:
We may want to make this configurable as well. Can we add `STORAGE_LOCAL_ROOTDIR` to the env.open namespace in values.yaml and just make the default `/storage`?

Reply:
I prefer to hard code local storage path.
I can not get any use case when we need it be configured.

Reply:
Ok, let’s leave for now

Reply:
Done